platforms/esp32/flash_monitor.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# TODO: comments & error handling
+
+set -e
+
+platforms/esp32/idf.py -p "$1" -b 921600 flash
+tio -b 115200 "$1"
```

Comment:
Add some sort of an "install guideline"' script for tools like tio, esptool and so on. Maybe not the IDF itself as it's… special.

Reply:
This is mostly an internal tool for me. Build and internal development instructions will be added to the repo after EW conference.